src/test/java/ladder/domain/LadderTest.java

```diff
@@ -0,0 +1,23 @@
+package ladder.domain;
+
+import ladder.factories.LadderLineFactory;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LadderTest {
+    @Test
+    public void create() {
+        final List<LadderLine> ladderLines = Arrays.asList(
+                LadderLineFactory.from(5),
+                LadderLineFactory.from(5),
+                LadderLineFactory.from(5),
+                LadderLineFactory.from(5)
+        );
+
+        assertThat(new Ladder(ladderLines)).isEqualTo(new Ladder(ladderLines));
```

Comment:
생성로직 검증에서 actual과 expected가 항상 같은 값이니 항상 통과하는 테스트가 됩니다. 어떤 경우에도 통과하는 테스트가 많아진다면 오히려 테스트에 대한 신뢰성이 떨어지지 않을까요?

이런 경우 2가지 고민을 해볼 수 있겠습니다.
1. 생성후에 상태를 확인할수 없는 경우에 테스트가 꼭 필요할까? (이미 의존하고 있는 LadderLineTest에서 다 진행하지 않았을까?)
2. 생성하는 동안에 어떠한 예외사항도 발생하지 않는 경우를 단언하고 싶다면 아래와 같이 표현할 것 같습니다.
```suggestion
        final List<LadderLine> ladderLines = Arrays.asList(
                LadderLineFactory.from(5),
                LadderLineFactory.from(5),
                LadderLineFactory.from(5),
                LadderLineFactory.from(5)
        );

       assertThatCode(() -> new Ladder(ladderLines))
                .doesNotThrowAnyException();
```